server/shared/src/main/scala/org/http4s/server/websocket/WebSocketBuilder2.scala

```diff
@@ -44,6 +45,7 @@ sealed abstract class WebSocketBuilder2[F[_]: Applicative] private (
     onHandshakeFailure: F[Response[F]],
     onClose: F[Unit],
     filterPingPongs: Boolean,
+    defragFrame: Boolean,
```

Comment:
You can add a deprecated constructor with the old signature, that sets this value to `false` by default.

Reply:
It looks like I passed the binary compatibility test. Thank you as always!